### DISCUSSION TITLE
Is the Q-value the output of the neural network?

### DISCUSSION THREAD

**Discussion by ec-ho-ra-mos** [vdemichev/DiaNN#479] [Q&A]
I understand that the neural network in DIA-NN is used to score the peptide IDs. The output of the neural network has the shape [1]. Is this the Q-value? If no, how can we derive the Q-value from the NN's output?

**Comment by vdemichev** [comment]
Q.Values in the main DIA-NN report are those calculated based on CScore values, which are what is produced by NNs.
